content/news/release-20170527.md

```diff
@@ -0,0 +1,80 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+We are today pushing a new bugfix release that addresses a collection of issues that were missed during the playtesting phase of release-20170421.
+
+The main changes in this hotfix are:
+
+* Disabled HiDPI support on Windows, fixing the graphical glitches this caused on non-HiDPI displays.
+* Fixes cursor glitches introduced with the new Mono 5.0 runtime.
```

Comment:
You're using the past tense above ("disabled"), but present here. I'd suggest changing the "fixes" here and below to "fixed".